spec/workflow/states/task_spec.rb

```diff
@@ -307,8 +307,7 @@
         it "retries if that error is raised" do
           # 1 regular run + 2 retries = 3 times
           3.times { expect_run_async(input, :error => "States.Timeout") }
-
-          workflow.run_nonblock
+          3.times { |i| Timecop.travel(Time.now.utc + (i * 10)) { workflow.run_nonblock } }
```

Comment:
The default IntervalSeconds value is 1, so now that we're actually checking this we can't just run_nonblock and expect it to execute each state right away.